Week2/week2_homework/w2_4.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const fav_books = [
+  {
+    title: "One Hundred Years of Solitude",
+    author: "Gabriel Garcia Marquez",
+    alreadyRead: false
+  },
+  {
+    title: "Ulysses",
+    author: "James Joyce",
+    alreadyRead: false
+  },
+  {
+    title: "Great Expectations",
+    author: "charles Dickens",
+    alreadyRead: true
+  }
+]
+
+fav_books.forEach((book) => {
+  console.log(book.title + " by " + book.author);
+  if (book.alreadyRead === true) {
+    console.log(`You already read ${book.title}`);
+  } else {
+    console.log(`You still need to read ${book.title}`);
+  }
+});
```

Comment:
👍 Fantastic use of `forEach()` and arrow functions! You'll get around to learning more about them in JS2.